builtin/v9/account/methods.go

```diff
@@ -11,5 +11,5 @@ var Methods = map[uint64]builtin.MethodMeta{
 	1: {"Constructor", *new(func(*address.Address) *abi.EmptyValue)},                   // Constructor
 	2: {"PubkeyAddress", *new(func(*abi.EmptyValue) *address.Address)},                 // PubkeyAddress
 	3: {"AuthenticateMessage", *new(func(*AuthenticateMessageParams) *abi.EmptyValue)}, // AuthenticateMessage
-	uint64(builtin.UniversalReceiverHookMethodNum): {"UniversalReceiverHook", *new(func(*[]byte) *abi.EmptyValue)}, // UniversalReceiverHook
+	uint64(builtin.UniversalReceiverHookMethodNum): {"UniversalReceiverHook", *new(func(*abi.EmptyValue) *abi.EmptyValue)}, // UniversalReceiverHook
```

Comment:
I think `CborBytes` would be _better_ but still not quite right since (I think) it's a not a requirement that these params are CBOR-able.